### DISCUSSION TITLE
HYP3 save_kmz.py

### DISCUSSION THREAD

**Discussion by GabiZuccolotto** [insarlab/MintPy#731] [Q&A]
Hello! I apologize if this has already been answered somewhere on the forum. I have just run a MintPy analysis with a stack of unwrapped interferograms generated by HYP3. I can view the results just fine using view.py and tsview.py, however, when I try to save the velocity results as a KMZ the file cannot be opened in Google Earth. I assume this has something to do with the geocoding of the results. Is there a work around to this problem? Thanks!

**Comment by yunjunz** [comment]
This is because HyP3 is using UTM coordinates while `save_kmz.py` only supports lat/lon currently, as explained in https://github.com/insarlab/MintPy/issues/686. 

There is no command-line ready workaround solution, unfortunately. 

However, `save_kmz.py` only takes `X/Y_FIRST/STEP` from the file, which can be easily converted from UTM to lat/lon using [`ut.to_latlon()`](https://github.com/insarlab/MintPy/blob/d219448d5878cfbc9ab9464c069e47fb586a71ec/mintpy/utils/utils0.py#L290) function, so this functionality should be easily added. Contributions are welcomed.
